src/cmds/qsub.c

```diff
@@ -6218,8 +6225,8 @@ main(int argc, char **argv, char **envp) /* qsub */
 	basic_envlist = job_env_basic();
 	if (basic_envlist == NULL)
 		exit_qsub(3);
-	if (V_opt)
-		qsub_envlist = env_array_to_varlist(envp);
+	
+	qsub_envlist = env_array_to_varlist(envp);
```

Comment:
Calling env_array_to_varlist in every condition with definitely decrease performance of qsub even with out -V because env_array_to_varlist is little heavy function which traverse whole environ char by char.

Reply:
I agree with your statement that it will affect performance of qsub.
The other alternative solution to this problem was to connect to the server at the very starting, before it was loading value to qsub_envlist in the "main" function, and fetch the default qsub arguments from the server. But this renders the use of a daemon pointless. 
Although this result in performance degradation but it is the only (valid) solution for this bug. 
If needed, we can enforce that -V option never be set in default arguments to ensure performance.